docs/study-def-tricks.md

```diff
@@ -319,4 +319,63 @@ study = StudyDefinition(
     ),
     **common_variables
 )
-```
\ No newline at end of file
+```
+
+## Extracting ethnicity across multiple time points
+
+Querying ethnicity can take a long time as a result of having to map from individual codes to broad categories. This is ok if you only extract ethnicity at one time point, but if you are running your study definition across multiple time points it can slow down the cohort extraction. As ethnicity is generally expected to remain constant over time, you can instead extract ethnicity once (taking the latest recorded ethnicity code for each patient) and then map these to the files generated by your main study definition for each time point.
```

Comment:
are there good examples for  "individual codes" and "broad categories" that we could add here?

Querying ethnicity can take a long time as a result of having to map from individual codes (e.g., XX) to broad categories (e.g., YY).

Reply:
I'll add this in